docs/content/staking/custody-providers.mdx

```diff
@@ -37,7 +37,7 @@ Coinlist is operating FLOW nodes and will automatically delegate your locked FLO
 
 #### Withdrawing Rewards
 
-If you would like to withdraw your rewards, you'll need to contact Coinlist's support team (team@coinlist.co).
+If you would like to withdraw your rewards, you'll need to contact Coinlist's support team [coinlist team](mailto:team@coinlist.co).
```

Comment:
This solution doesn't work, when clicking on the link 404 page is returned. Check it out in this preview - https://flow-docs-staging.fly.dev/nodes/staking/custody-providers?preview=mailto-link-format

Reply:
All mailto links are broken at the moment, This PR is to fix the format so that when mailto links are processed correctly this link will work.

Reply:
Formatting changes approved. Still needs fix to the underlying problem with 404.